jobs/config.json

```diff
@@ -3640,6 +3640,28 @@
       "sig-cli"
     ]
   },
+  "ci-kubernetes-e2e-gce-stable2-stable1-gpu-upgrade": {
+    "args": [
+      "--check-leaked-resources",
+      "--check-version-skew=false",
+      "--env=STORAGE_MEDIA_TYPE=application/vnd.kubernetes.protobuf",
```

Comment:
Why is this needed?

Reply:
For upgrade test, I think one needs to specify STORAGE_MEDIA_TYPE for etcd. I have used STORAGE_MEDIA_TYPE=application/json in my manual runs of upgrade test to make it work, but looks like STORAGE_MEDIA_TYPE=application/vnd.kubernetes.protobuf has been used in the existing ci-kubernetes-e2e-gce-stable2-stable1-upgrade-cluster tests.